app/views/early_career_payments/claims/eligibility_confirmed.html.erb

```diff
@@ -0,0 +1,61 @@
+<% content_for(:page_title, page_title("You are eligible to claim for early-career payment", policy: current_policy_routing_name)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">You are eligible <%= I18n.t("early_career_payments.claim_description") %></h1>
+    <p class="govuk-body">
+      Based on the answers you have provided you will be able to claim £5,000.
+      This could increase to £7,500 if the school you teach at is considered
+      to be in an uplift area.
+    </p>
+
+    <p>
+      <%= link_to "Uplift areas", "#{EarlyCareerPayments.eligibility_page_url}#local-authorities-with-uplift-payments", class: "govuk-link" %>
```

Comment:
Not 100% sure this is the same eligibility page url as referenced elsewhere in the application. Could you have a look at what the M+P & TSLR ones are and reach out to Laura/Muhammad about this please. Not a blocker, but something to be aware of.

Reply:
Looking at the code in the Prototype V7 on the link there is are `target` and `title` attributes. Can you get these added to please. Please check with Brian to confirm, I think we use `title` for assistive technologies.